sample/src/androidTest/java/com/schibsted/spain/barista/sample/ViewPager2Test.kt

```diff
@@ -0,0 +1,149 @@
+    assertDisplayed("2")
+  }
+
+  @Test
+  fun checkHorizontalSwipeForward_withId() {
+    setViewPager(orientation = HORIZONTAL)
+    swipeViewPagerForward(R.id.viewPager2)
+    assertDisplayed("2")
+  }
+
+  @Test
+  fun checkHorizontalSwipeBack() {
+    setViewPager(orientation = HORIZONTAL)
+    swipeViewPagerForward()
+    swipeViewPagerBack()
+    assertDisplayed("1")
+  }
+
+  @Test
+  fun checkHorizontalSwipeBack_withId() {
+    setViewPager(orientation = HORIZONTAL)
+    swipeViewPagerForward(R.id.viewPager2)
+    swipeViewPagerBack(R.id.viewPager2)
+    assertDisplayed("1")
+  }
+
+  @Test
+  fun checkVerticalSwipeForward() {
+    setViewPager(orientation = VERTICAL)
+    swipeViewPagerForward()
+    assertDisplayed("2")
+  }
+
+  @Test
+  fun checkVerticalSwipeForward_withId() {
+    setViewPager(orientation = VERTICAL)
+    swipeViewPagerForward(R.id.viewPager2)
+    assertDisplayed("2")
+  }
+
+  @Test
+  fun checkVerticalSwipeBack() {
+    setViewPager(orientation = VERTICAL)
+    swipeViewPagerForward()
+    swipeViewPagerBack()
+    assertDisplayed("1")
+  }
+
+  @Test
+  fun checkVerticalSwipeBack_withId() {
+    setViewPager(orientation = VERTICAL)
+    swipeViewPagerForward(R.id.viewPager2)
+    swipeViewPagerBack(R.id.viewPager2)
+    assertDisplayed("1")
+  }
+
+  @Test
+  fun swipingForwardHorizontallyWhileAtTheLastPageDoesNotCrash() {
+    setViewPager(orientation = HORIZONTAL)
+    swipeViewPagerForward()
+    swipeViewPagerForward()
+    assertDisplayed("2")
+  }
+
+  @Test
+  fun swipingForwardHorizontallyWhileAtTheLastPageDoesNotCrash_withId() {
+    setViewPager(orientation = HORIZONTAL)
+    swipeViewPagerForward(R.id.viewPager2)
+    swipeViewPagerForward(R.id.viewPager2)
+    assertDisplayed("2")
+  }
+
+  @Test
+  fun swipingBackHorizontallyWhileAtTheFirstPageDoesNotCrash() {
+    setViewPager(orientation = HORIZONTAL)
+    swipeViewPagerBack()
+    assertDisplayed("1")
+  }
+
+  @Test
+  fun swipingBackHorizontallyWhileAtTheFirstPageDoesNotCrash_withId() {
+    setViewPager(orientation = HORIZONTAL)
+    swipeViewPagerBack(R.id.viewPager2)
+    assertDisplayed("1")
+  }
+
+  @Test
+  fun swipingForwardVerticallyWhileAtTheLastPageDoesNotCrash() {
+    setViewPager(orientation = VERTICAL)
+    swipeViewPagerForward()
+    swipeViewPagerForward()
+    assertDisplayed("2")
+  }
+
+  @Test
+  fun swipingForwardVerticallyWhileAtTheLastPageDoesNotCrash_withId() {
+    setViewPager(orientation = VERTICAL)
+    swipeViewPagerForward(R.id.viewPager2)
+    swipeViewPagerForward(R.id.viewPager2)
+    assertDisplayed("2")
+  }
+
+  @Test
+  fun swipingBackVerticallyWhileAtTheFirstPageDoesNotCrash() {
+    setViewPager(orientation = VERTICAL)
+    swipeViewPagerBack()
+    assertDisplayed("1")
+  }
+
+  @Test
+  fun swipingBackVerticallyWhileAtTheFirstPageDoesNotCrash_withId() {
+    setViewPager(orientation = VERTICAL)
+    swipeViewPagerBack(R.id.viewPager2)
+    assertDisplayed("1")
+  }
+
+  /** Helper function */
```

Comment:
I would remove the comment. But it's not an stopper to approve the PR, of course 😄